.dependabot/config.yml

```diff
@@ -12,3 +12,7 @@ update_configs:
       - match:
           dependency_name: "io.dropwizard:*"
           version_requirement: "2.+"
+      - match:
+          # We currently use the latest version supported by Dropwizard v1, so cannot upgrade
```

Comment:
Not entirely confident that the YAML parser will be happy with putting a comment before the object fields below, but let's give it a shot.

Reply:
I gave it to their [validator](https://dependabot.com/docs/config-file/validator/), and it seemed happy.